packages/eslint-plugin-stories/README.md

```diff
@@ -47,9 +45,10 @@ or configure the rules individually
 
 ## Rules
 
-Name                           | Description                                                                                                | Config
------------------------------- | ---------------------------------------------------------------------------------------------------------- | -----------
-csf-object-literal-or-function | Enforce that stories are either an object literal or function.                                             | recommended
-csf-v3-types                   | Enforce component story format (CSF) v3 stories have explicit TypeScript types where needed for inferrence | strict
-no-csf-v2                      | Use object stories (component story format v3) instead of functions.                                       | strict
-no-ext-resources-in-stories    | Prevent external resources from being loaded in stories.                                                   | recommended
+| Name                            | Description                                                                                                | Config      |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------------- | ----------- |
+| csf-object-literal-or-function  | Enforce that stories are either an object literal or function.                                             | recommended |
+| csf-v3-types                    | Enforce component story format (CSF) v3 stories have explicit TypeScript types where needed for inferrence | strict      |
+| no-csf-v2                       | Use object stories (component story format v3) instead of functions.                                       | strict      |
+| no-ext-resources-in-stories     | Prevent external resources from being loaded in stories.                                                   | recommended |
+| no-new-components-without-story | Enforce that new components are created with a corresponding story                                         | recommended |
```

Comment:
Is it misleading to be calling this rule `no-new-components-without-story`? I think we should remove the word, "new" from the documentation in this repository and code. ESLint is a static analysis tool and does not know the difference between "new" and "old", the rule just flags components without stories

Reply:
That's a great point